### PR TITLE
JBPM-6802:Add breadcrumbs to list and details views

### PR DESCRIPTION
**Pull request by nmirasch**

### CODE REVIEW

**Comment by nmirasch** [comment]
@cristianonicolai could you take a look on this code, to see if it's the right way? Thanks!!!

File: jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/list/BreadcrumbManagementImpl.java [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+                                      }
+                                  });
+
+    }
+
+    public void addEndBreadcrumb(String perspectiveId,
+                                  String label) {
+        breadcrumbs.addBreadCrumb(perspectiveId,
+                                  label,
+                                  new Command() {
```
Comment (cristianonicolai) [inline jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/list/BreadcrumbManagementImpl.java]: @nmirasch what if you pass null for the command?

File: jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/main/java/org/jbpm/workbench/es/client/editors/errordetails/ExecutionErrorDetailsPresenter.java [hunk 2]
```diff
@@ -60,14 +62,17 @@
...
+
     @WorkbenchPartView
     public UberElement<ExecutionErrorDetailsPresenter> getView() {
         return view;
     }
 
     @DefaultPosition
     public Position getPosition() {
-        return CompassPosition.EAST;
+        return CompassPosition.SELF;
```
Comment (cristianonicolai) [inline jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/main/java/org/jbpm/workbench/es/client/editors/errordetails/ExecutionErrorDetailsPresenter.java]: @nmirasch not sure why we need to change this now?

File: jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskdetailsmulti/TaskDetailsMultiPresenter.java [hunk 3]
```diff
@@ -151,6 +162,18 @@ public void onTaskSelectionEvent(@Observes final TaskSelectionEvent event) {
         containerId = event.getContainerId();
         processId = event.getTaskName();
 
+        breadcrumbs.createListDetailBreadcrumbs(placeManager,
```
Comment (cristianonicolai) [inline jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskdetailsmulti/TaskDetailsMultiPresenter.java]: so creating both admin and regular task list, I'm wondering if we should consider breaking these screens into two?

File: jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/main/java/org/jbpm/workbench/es/client/editors/errordetails/ExecutionErrorDetailsPresenter.java [hunk 4]
```diff
@@ -60,14 +62,17 @@
     @Inject
     private Caller<ExecutorService> executorServices;
 
+    @Inject
+    BreadcrumbManagement breadcrumbs;
```
Comment (cristianonicolai) [inline jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/main/java/org/jbpm/workbench/es/client/editors/errordetails/ExecutionErrorDetailsPresenter.java]: @nmirasch  can we move this to a more generic location (AbstractMultiGridPresenter)? As well as the breadcrumb related methods, that way its easier to maintain any related changes.

**Review by cristianonicolai — CHANGES_REQUESTED** [review]
@nmirasch overall looks good, take a look in some comments I made. Also check some code formatting :)

**Comment by nmirasch** [comment]
@cristianonicolai I changed a bit the breacrumb integration, centralizing it at list presenters and their items selection actions. I think It looks simpler to maintain. Could you take a look? Thanks!

**Review by cristianonicolai — APPROVED** [review]

**Pull request merged**